test/e2e-kuttl/quota-forest/04-install.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: test
   labels:
     quota_context: "gold"
-    quota_service: "service-root"
+    quota_service: "gold"
```

Comment:
why was this change necessary for all tests ?

Reply:
The new addition of a `default` node mandated that we have a child in the `quota-service` tree besides just having the root node. To match the same structure already present in the `quota-context` tree, then the service tree was updated with the a `gold` child. The `gold` child can be named anything, so the name was just arbitrary.